docker-storage-setup.1

```diff
@@ -98,42 +94,42 @@ WIPE_SIGNATURES:
       overwritten as suitable. This also means that if there is any
       data on disk, it will be lost.
 
-DOCKER_ROOT_VOLUME:
+CONTAINER_ROOT_LV_MOUNT_PATH:
```

Comment:
CONTAINER_ROOT_LV_MOUNT_PATH is already there. I guess you should modify that section.